<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefPlaylist</class>
 <widget class="QWidget" name="PrefPlaylist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>467</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="playlist_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Playlist</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Playback</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="play_from_start_check">
            <property name="text">
             <string>P&amp;lay files from start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="play_on_load_check">
            <property name="text">
             <string>S&amp;tart playback after loading a playlist</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="play_next_automatically_check">
            <property name="text">
             <string>Pla&amp;y next file automatically</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ignore_errors_check">
            <property name="text">
             <string>Ig&amp;nore playback errors</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Adding files</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="auto_add_to_playlist_check">
            <property name="text">
             <string>&amp;Add files to the playlist automatically</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="media_to_add_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Add files from &amp;folder:</string>
              </property>
              <property name="buddy">
               <cstring>media_to_add_combo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="media_to_add_combo">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="recursive_check">
            <property name="text">
             <string>Add files in directories &amp;recursively</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="getinfo_check">
            <property name="text">
             <string>Get &amp;info automatically about files added (slow)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dockable_check">
         <property name="text">
          <string>The playlist window is &amp;dockable</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="misc_tab">
      <attribute name="title">
       <string>&amp;Misc</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="autosort_check">
         <property name="text">
          <string>A&amp;uto sort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="case_sensitive_search_check">
         <property name="text">
          <string>Cas&amp;e sensitive search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autosave_on_exit_check">
         <property name="text">
          <string>&amp;Save a copy of the playlist on exit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="allow_delete_files_check">
         <property name="text">
          <string>Enable the option to delete files from &amp;disk</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>301</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>playlist_tab</tabstop>
  <tabstop>play_from_start_check</tabstop>
  <tabstop>play_on_load_check</tabstop>
  <tabstop>play_next_automatically_check</tabstop>
  <tabstop>ignore_errors_check</tabstop>
  <tabstop>auto_add_to_playlist_check</tabstop>
  <tabstop>media_to_add_combo</tabstop>
  <tabstop>recursive_check</tabstop>
  <tabstop>getinfo_check</tabstop>
  <tabstop>dockable_check</tabstop>
  <tabstop>autosort_check</tabstop>
  <tabstop>case_sensitive_search_check</tabstop>
  <tabstop>autosave_on_exit_check</tabstop>
  <tabstop>allow_delete_files_check</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>auto_add_to_playlist_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>media_to_add_combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_add_to_playlist_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>media_to_add_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
